demos/python_demos/README.md

```diff
@@ -0,0 +1,9 @@
+## Prepare OVMS image with Python support and Optimum package installed
+
+From the root of the repository run:
```

Comment:
This step is common for all the 3 demos but it might be missed if someone started reading directly form a demo
It might be safer to repeat this step in each demo.
On this page could just just a very short intro and a list of demos.

Reply:
I suppose we would also need to add git clone of the repo to each demo, so they can get auto tested.